src/main/kotlin/org/malachite/estella/services/JobSeekerService.kt

```diff
@@ -34,6 +36,22 @@ class JobSeekerService(
             throw UserAlreadyExistsException()
         }
 
+    fun deleteJobSeeker(id: Int, jwt: String?) =
+        if(!checkRights(id, jwt).contains(Permission.DELETE)) throw UnauthenticatedException()
+        else deleteJobSeeker(id)
```

Comment:
O ile ogarniam to ten else jest niepotrzebny, bo wyjątek spowoduje że dalsza część nie zostanie wykonana

Reply:
Jak pisałeś gdzieś wcześniej, tu jest =, więc ten else musi być